.github/workflows/stale.yml

```diff
@@ -0,0 +1,24 @@
+name: Close Stale Pull Requests
+
+on:
+  schedule:
+    #Runs at 00:00 every day
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+
+    steps:
+     - uses: actions/checkout@master
+     - name: stale
+       uses: gatsbyjs/stale@master
+       with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          DAYS_BEFORE_STALE: 30
+          DAYS_BEFORE_CLOSE: 7
+          STALE_ISSUE_LABEL: 'stale'
+          STALE_PR_LABEL: 'stale'
+          STALE_PR_MESSAGE: 'Hi, were are going to mark this PR as stale because it has not had any updates for 30 days. If no further activity occurs within the following 7 days, it will be automatically closed so that others can take up the issue.If you are still working on this PR, please make a follow-up commit within 7 days and do leave a comment if you are still working on this PR. Please also let us know if you are stuck so we can help you!'
```

Comment:
we are going to
inactivity for 30 days?
If you are still working on this PR, please make a follow-up commit within next 7 days and leave a comment to remove the stale label.